doc/man3/SSL_SESSION_get0_hostname.pod

```diff
@@ -51,12 +51,21 @@ SSL_SESSION_set1_hostname() returns 1 on success or 0 on error.
 
 SSL_SESSION_set1_alpn_selected() returns 1 on success or 0 on error.
 
+=head1 NOTES
+
+SSL_SESSION_get0_hostname() is not restricted by the negotiated TLS version. In
+both TLSv1.2 and TLSv1.3, it returns the Server Name Indication (SNI) value
+stored in the session if one was accepted by the server; otherwise it returns
```

Comment:
I am puzzled why anything needs to be said about lack of dependence on the TLS version. If there was such a dependence, with the function not working for either TLS 1.2, or TLS 1.3, then perhaps a note would be needed. But otherwise, I don't see any reason to call out this particular non-dependence.  There are many other things that that don't influence availability of the SNI name (if the extension was acknowledged/accepted by the server).

The next sentence is a useful reference to a related function.